.github/CHANGELOG.md

```diff
@@ -10,10 +10,15 @@
 
 ### Bug fixes
 
+* Change the name of `qml.Interferometer` to `qml.InterferometerUnitary` due to PennyLane deprecation
```

Comment:
Maybe this is better placed in `Improvements`? :thinking: It doesn't feel like a bug fix (as it's a change due to intended behaviour), but also not like any of the other categories :smile: